corehq/apps/hqwebapp/two_factor_gateways.py

```diff
@@ -145,6 +145,8 @@ def _check_for_exceeded_rate_limits(ip, num, user):
         if status == _status_accepted:
             _report_usage(ip_address, number, username)
         else:
+            window = window or 'unknown'
+            status = status or 'unknown'
```

Comment:
My preference might be to move this into the `RateLimitedTwoFactorLog.objects.create` call below, so that it doesn't affect the value of the variable, and it's clear that the point is just to make sure that the Log object never has a null value there

Reply:
Makes sense to me. Added it to the `save` method on the RateLimitedTwoFactorLog object